crates/re_ui/src/lib.rs

```diff
@@ -366,6 +366,24 @@ impl ReUi {
         response
     }
 
+    #[allow(clippy::unused_self)]
+    pub fn checkbox(
+        &self,
+        ui: &mut egui::Ui,
+        selected: &mut bool,
+        text: impl Into<egui::WidgetText>,
+    ) -> egui::Response {
+        ui.scope(|ui| {
+            ui.visuals_mut().widgets.hovered.expansion = 0.0;
+            ui.visuals_mut().widgets.active.expansion = 0.0;
+            ui.visuals_mut().widgets.open.expansion = 0.0;
+
+            // note: fully qualified syntax to dodge script/lint.py
```

Comment:
We already have a `NOLINT` marker, but here it's of tricky as `ui.checkbox` is multiline. This makes me realise that this wouldn't even be picked up currently, because of the non-whitespace comment inside the `ui.checkbox`:
```rust
ui  // hello
  .checkbox(...)
  .checked()
```
I'm trying very much to avoid a `syn`/`cargo script` kind of rabbit hole here, but I have yet to identify a satisfactory shortcut.

Reply:
Well, I *did* dip my toes in that rabbit hole 😅 Luckily, it ended up putting me on the path for a more reasonable fix...